### DISCUSSION TITLE
How to use Telegram notifications?

### DISCUSSION THREAD

**Discussion by Falci** [monicahq/monica#7212] [Q&A]
I see some config about Telegram, but not sure how to proper use it.

https://github.com/monicahq/monica/blob/6268c0f78dd1c2f5a1644bc784731bba793c8c8c/.env.example#L94-L97

**Comment by despokd** [comment]
Did you follow the steps from the docs? https://docs.monicahq.com/developers/setup-telegram

**Reply by Falci** [reply]
Can't make it work.
How to test?

I tried to access the custom URL, but it returns 404. 
If I send a message to the bot (regular, via telegram) I can see telegram hits the endpoint with a POST request. But it returns 404
